draft-irtf-cfrg-rsa-blind-signatures.md

```diff
@@ -338,6 +352,119 @@ Steps:
    raise "invalid signature" and stop
 ~~~
 
+## External Application Interface {#salted-interface}
+
+This section presents an application interface for blinding, finalizing, and verifying
+messages that is build on the internal functions described in {{generation}}. This
+interface injects additional entropy into application messages by choosing a random
+salt of length 32 bytes, prepending the salt to the input message, and then invoking
+the internal functions in {{generation}}. Note that this only changes what is passed
+to rsabssa_blind and rsabssa_finalize, as the application message is not provided as
+input to rsabssa_blindsign.
+
+Applciations that provide high-entropy input messages can expose the internal
+rsabssa_blind and rsabssa_finalize directly, as the additional message randomization
+does not offer security advantages. See {{apis}} and {{message-entropy}} for more information.
+
+### Salted Blind
+
+rsabssa_salted_blind invokes rsabssa_blind with a salted input message and outputs the
+blinded message to be sent to the server and the corresponding inverse, both encoded
+as octet strings, as well as the fresh message salt, which is 32 random bytes.
```

Comment:
Mixing bytes and octets in the same sentence. What's the reasoning for mixing both nomenclatures?

Reply:
We already follow this mixed convention this in the draft, so if we're going to address this let's do it separately.